src/main/java/org/semver4j/Semver.java

```diff
@@ -515,14 +535,14 @@ public String toString() {
     }
 
     /**
-     * The types of diffs between two versions.
+     * The types of diffs between two versions. The higher the ordinal value of the enum is, the greater is the diff.
```

Comment:
Here should be added an information to keep enums in the order.

Reply:
You mean beyond the sentence that I added? Isn't it clear from that already?

Reply:
Maybe it is, but I'm worry when someone change the order. But maybe added test will be enough.